src/components/AudioButton.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import AudioButton from './AudioButton'
+
+export default {
+  title: 'Components/Audio Button',
+  component: AudioButton,
+}
+
+const Template = (args) => <AudioButton {...args} />
+
+export const Primary = Template.bind({})
```

Comment:
How does that work? 😲 Crazy advanced stuff!

Reply:
storybook ;-)